CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Change log for Microsoft365DSC
 
-#1.21.1208.1
+# 1.21.1215.1
+
+* AADConditionalAccessPolicy
+  * Switched to the beta endpoint to allow the export to capture policies
+    with device compliance conditions configured;
+* EXOAntiphishPolicy
+  * Fixed an issue where trying to create a new policy would result in its
+    Identity being set to System.Collections.Hashtable;
+    FIXES #1620
```

Comment:
Why is this in here?

Reply:
Because it fixes that issue. We've always listed changes that are related to a specific issue that way in the past.